build/evergreen_config_lib/variants.py

```diff
@@ -179,15 +189,10 @@ def days(n):
              '.debug-compile .stdflags !.c89',
              '.debug-compile !.sspi .openssl',
              '.debug-compile !.sspi .nossl',
-             '.authentication-tests .openssl',
-             '.authentication-tests .valgrind',
```

Comment:
The `latest` tasks make sense to remove on this Ubuntu 16.04 variant, since there are no longer latest server builds published for Ubuntu 16.04.

The `authentication-tests` use the run-auth-tests.sh script, which connect to external servers (e.g. Atlas dev instances). I think those should ideally be passing, but they are not on the waterfall. Unless we know why, I would vote we leave them to investigate in case it indicates a bug.